librb/include/rb_include.h

```diff
@@ -0,0 +1,44 @@
+#ifndef _RB_INCLUDE_H
+#define _RB_INCLUDE_H 1
+
+// This is exactly how they recommend doing it in the docs by Alan Smithee,
+// but the gcc docs on the other hand, with tongue in cheek, tell the tale
+// by metaphor: https://gcc.gnu.org/onlinedocs/cpp/Computed-Includes.html
+
+#include <RB_INC_ASSERT_H
```

Comment:
Hmm, seems kinda hacky.  On the other hand, the other way is a hack too.  `/dev/null` is guaranteed by POSIX, so I guess that is fine.

Do you change the configure script to appropriately handle this situation?